### PR TITLE
Update docs to reflect Imaging moving out of Prerelease.

### PR DESCRIPTION
**Pull request by samgannawayA**
## Description
Drop "_beta" from Imaging's module name.
Update example script and plugin.
Update SolidColor after fixes in January.
Clean up various pages, including filing interfaces under Objects.

### CODE REVIEW

File: src/pages/ps_reference/changelog/index.md [hunk 1]
```diff
@@ -6,7 +6,29 @@ description: Contains a running log of changes to the UXP API environment in Ado
 
 # Photoshop API Changelog
 
-## Photoshop Beta (24.2 February 2023)
+## Photoshop Beta (24.2 February 2022)
```
Comment (dereknnovo) [inline src/pages/ps_reference/changelog/index.md:9]: wrong date

File: src/pages/ps_reference/media/photoshopcore.md [hunk 2]
```diff
@@ -110,7 +110,51 @@ based on embedded color profiles.
 | Name | Type |
 | :------ | :------ |
 | `sourceColor` | ColorDescriptor |
-| `targetModel` | ColorConversionModel |
+| `targetModel` | ColorConversionModel.RGB |
+
+LabColorDescriptor
+
+Convert to LaB
```
Comment (heewooa) [inline src/pages/ps_reference/media/photoshopcore.md:117]:
```suggestion
Convert to Lab
```

**Review by heewooa — APPROVED** [review]
Please address the date/version in changelog, otherwise looks fine

File: src/pages/ps_reference/changelog/index.md (hunk 1 above)
Comment (samgannawayA) [reply src/pages/ps_reference/changelog/index.md]: Not just that but also drop the "Beta".

File: src/pages/ps_reference/media/photoshopcore.md (hunk 2 above)
Comment (samgannawayA) [reply src/pages/ps_reference/media/photoshopcore.md]: That's fair.  It is "Lab" in the UI.

**Pull request merged**